static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -54,77 +55,79 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
 
   return (
     <Fragment>
-      <FeedbackItemHeader eventData={eventData} feedbackItem={feedbackItem} />
-      <OverflowPanelItem ref={overflowRef}>
-        <Section>
-          <MessageSection eventData={eventData} feedbackItem={feedbackItem} />
-        </Section>
-
-        {!crashReportId || (crashReportId && url) ? (
-          <Section icon={<IconLink size="xs" />} title={t('URL')}>
-            <TextCopyInput
-              style={urlIsLink ? {color: theme.blue400} : undefined}
-              onClick={
-                urlIsLink
-                  ? e => {
-                      e.preventDefault();
-                      openNavigateToExternalLinkModal({linkText: displayUrl});
-                    }
-                  : () => {}
-              }
-            >
-              {displayUrl}
-            </TextCopyInput>
+      <SurfaceProvider value="details">
```

Comment:
Seems like a more clear interface would be to have the prop be `name` not `value`